### DISCUSSION TITLE
Client host rejected: Access denied

### DISCUSSION THREAD

**Discussion by ngms17** [jertel/elastalert2#659] [Q&A] [answered]
Getting this error when trying to send an email. I am using my organization smtp host:

ERROR:root:Error while running alert email: Error connecting to SMTP host: (554, b'5.7.1 <suricata-elk01.domain.tld[x.x.x.x]>: Client host rejected: Access denied')

**Comment by nsano-rururu** [comment, marked as answer]
Ask your organization's operations administrator about the outbound settings.

**Reply by ngms17** [reply]
So this is most likely an issue with the mail servers and not on elastalert itself. Thank you!
